src/app/offers/offers-card/offers-card.component.html

```diff
@@ -5,37 +5,44 @@
+                      lg:min-h-[30rem]">
         </a>
         <app-offer-investment-info [offer]="offer" class="absolute float-left lg:top-6 left-6 lg:rounded-full
             -bottom-10 lg:bottom-auto rounded-2xl py-3 px-6 shadow-lg bg-white">
         </app-offer-investment-info>
       </div>
-      <div class="w-full p-6 bg-white rounded-b-4xl lg:rounded-none lg:rounded-r-4xl lg:w-5/12">
-        <h1 class="mt-6 lg:mt-0 text-3xl lg:text-4xl uppercase tracking-wider font-light leading-relaxed">
-          {{ offer.name }}
-        </h1>
-        <p class="my-2 font-light text-sm lg:text-base text-gray-500 leading-relaxed">
-          {{ offer.about }}
-        </p>
-        <app-funding-progress [campaign]="offer"></app-funding-progress>
+      <div class="flex flex-col justify-between w-full p-6 bg-white rounded-b-4xl lg:rounded-none lg:rounded-r-4xl
+                  lg:w-5/12">
+        <div>
+          <h1 class="mt-6 lg:mt-0 text-3xl lg:text-4xl uppercase tracking-wider font-light leading-relaxed">
+            {{ offer.name | truncate:40:true:false }}
+          </h1>
+          <p class="my-2 font-light text-sm lg:text-base text-gray-500 leading-relaxed">
+            {{ offer.about | truncate:400:true:false }}
+          </p>
+        </div>
 
-        <button class="bg-indigo-800 hover:bg-indigo-900 transition-all
-    w-full py-3 mt-6 text-white rounded-full flex items-center justify-center"
-                aria-label="Learn more"
-                routerLink="{{ offer.ansName | issuerPath }}">
-          Learn more
-          <svg class="ml-3" width="9" height="14" viewBox="0 0 9 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <path d="M1.5 1.1665L7.33333 6.99984L1.5 12.8332" stroke="white" stroke-width="2" stroke-linecap="round"
-                  stroke-linejoin="round"/>
-          </svg>
-        </button>
+        <div>
+          <app-funding-progress [campaign]="offer"></app-funding-progress>
+
+          <button class="bg-indigo-800 hover:bg-indigo-900 transition-all w-full py-3 mt-6 text-white rounded-full flex
+                         items-center justify-center"
+                  aria-label="Learn more"
+                  routerLink="{{ offer.ansName | issuerPath }}">
+            Learn more
+            <svg class="ml-3" width="9" height="14" viewBox="0 0 9 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <path d="M1.5 1.1665L7.33333 6.99984L1.5 12.8332" stroke="white" stroke-width="2" stroke-linecap="round"
+                    stroke-linejoin="round"/>
+            </svg>
+          </button>
+        </div>
       </div>
     </div>
 
     <div *ngIf="size === 'small'" class="w-full h-full flex flex-col justify-start shadow-xl hover:shadow-xl
-            transition-all duration-300 rounded-3xl">
-      <div class="relative w-full lg:h-52">
+                                         transition-all duration-300 rounded-4xl">
```

Comment:
There was a very slight misalignment of card content and its shadow on the bottom of the card. It may be kind of hard to see, look very closely at the bottom left and right rounded corners in the before and after images.
Before:
![Screenshot 2021-09-22 at 14 28 59](https://user-images.githubusercontent.com/12549625/134379830-9cca09d6-d81b-4f17-aa45-83475990aaef.png)
After:
![Screenshot 2021-09-22 at 14 30 29](https://user-images.githubusercontent.com/12549625/134379836-d2e60813-6f03-4607-b478-eb5b469248da.png)